paris/src/main/java/com/airbnb/paris/proxies/ViewProxy.java

```diff
@@ -35,6 +38,13 @@
 
     private static final int NOT_SET = -10;
 
+    public static final int PORTERDUFF_MODE_SRC_OVER = 3;
+    public static final int PORTERDUFF_MODE_SRC_IN = 5;
+    public static final int PORTERDUFF_MODE_SRC_ATOP = 9;
+    public static final int PORTERDUFF_MODE_MULTIPLY = 14;
+    public static final int PORTERDUFF_MODE_SCREEN = 15;
+    public static final int PORTERDUFF_MODE_ADD = 16;
```

Comment:
I just understood where these weird numbers are coming from. It's simply that all the modes aren't supported in XML. If you look at `PorterDuff.Mode` there are 17 total and they have a `nativeInt` field which corresponds to these numbers. Let's use those instead.